sdk/communication/communication-callingserver/package.json

```diff
@@ -68,7 +68,7 @@
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/communication-common": "^1.1.0",
+    "@azure/communication-common": "^1.1.1",
```

Comment:
The usage of `^` will ensure that the customer gets the latest version of the common package among all the versions that have major version 1. Any reason to explicitly change this?

Reply:
Calling server SDK is dependent on the changes in common SDK from feature/communication-CallingServer-preview branch which is building using 1.1.1 version. Keeping 1.1.0 version causing the changes from feature branch are not reflecting in CallingServer SDK. Hope my understanding is correct.

Reply:
Additional information: Currently we are validating calling server SDK APIs in the end samples we have been developing.